frontend/src/chrome/ButtonGroup.tsx

```diff
@@ -1,10 +1,12 @@
 import React from 'react'
 import { Link } from 'react-router-dom'
+import classNames from 'classnames'
 
 import Icon from './Icon'
 
 export interface ButtonGroupProps {
   items: ButtonGroupItem[]
+  currentPathname: string
```

Comment:
re-work this to an optional `selectedIndex?: number` prop & compare to the passed in `selectedIndex` prop to `i` in the map function instead of hard-coding the comparison property. set the default value to `-1` to have a number that won't match the index.

The call site will look something like this:

```tsx
// assume "buttons" is the array of button objects and "currentPathname" is the link string

return (<ButtonGroup
  items={buttons}
  selectedIndex={buttons.findIndex((b) => currentPathname === b.link)}
  >)
}
```

This prevents tight coupling